test/components/rankers/test_transformers_similarity.py

```diff
@@ -36,7 +36,7 @@ def test_to_dict_with_custom_init_parameters(self):
             scale_score=False,
             calibration_factor=None,
             score_threshold=0.01,
-            model_kwargs={"torch_dtype": "auto"},
+            model_kwargs={"torch_dtype": torch.float16},
```

Comment:
I think it would be great to also add a test for the `bnb_4bit_compute_dtype` here as well!

Reply:
done